website/content/docs/platform/k8s/helm/run.mdx

```diff
@@ -482,6 +482,28 @@ $ kubectl exec -ti <name of pod> -- vault operator unseal
 After a few moments the Vault cluster should elect a new active primary. The Vault
 cluster is now upgraded!
 
+#### Interaction with Raft Autopilot Automated Upgrades
+
+If `server.ha.raft` is enabled with Vault 1.11+, Autopilot has the ability to do
+[Automated Upgrades](/docs/concepts/integrated-storage/autopilot#automated-upgrades).
+If using the default `server.updateStrategyType` of `OnDelete` for the Vault StatefulSet, then
+the same upgrade procedure above will continue to work as expected,
+except that the leader will automatically switch over once the majority of pods
+are running the newer version of Vault.
+
+!> **IMPORTANT NOTE**: Use `OnDelete` instead of `RollingUpdate` for these automated upgrades,
+as `RollingUpdate` will kill the pods in an unpredictable order and could cause the cluster to
+upgrade and then downgrade the leader's version, which must be avoided.
+
+If you increase the number of replicas during the upgrade, be sure that **all** pods
```

Comment:
Is this the first time we mention increasing the number of replicas? It feels like this and the next paragraph are part of an entirely different strategy from the guide leading up to here. EDIT: More complete comment above